src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -153,6 +153,23 @@ private void recordNewSentEmail(String subject,
 
         moveJobToGroupModeTaskQueue();
         moveJobToAddressModeTaskQueue();
+
+        if (addressModeOn && groupModeOn) {
+            statusToAdmin = "Email will be sent within an hour to "
+                            + addressReceiverListString + "," + groupReceiverListFileKey;
+            statusToUser.add(new StatusMessage("Email will be sent within an hour to "
```

Comment:
I suggest to put the `statusToUser` thing in the `execute` method. It is not `recordNewSentEmail`'s responsibility to notify the user. Also, you can use ternary operator `? ... : ...` to get rid of `if-else` branch as your `if-else` branch is almost the same.